projects/samples/devices/controllers/laser_pointer/laser_pointer.c

```diff
@@ -25,20 +25,20 @@
 #include <webots/motor.h>
 #include <webots/robot.h>
 
-#define TIME_STEP 64
-
 int main() {
+
```

Comment:
ClangFormat test is failing because of this new line.
It should be removed to pass the tests.